cx_Freeze/macdist.py

```diff
@@ -324,6 +324,9 @@ def run(self):
                 self.copy_tree(resource, self.resourcesDir + '/' +
                                destination)
             else:
+                parent_dirs = os.path.dirname(self.resourcesDir + '/' +
```

Comment:
Any reason you didn't use os.path.join(self.resourcesDir, destination)?

Reply:
I think he kept the previous code. See, before and after it has similar code.

Reply:
Yeah. I see that. So should the other code be changed as well? There are two other occurrences. I guess since this is macOS specific code that it isn't strictly necessary. :-)

Reply:
Yes I maintained the current form. Would you like me to add a commit changing them all to os.join?

Reply:
May as well. Thanks. After that I'll merge it.

Reply:
Ok done, anything else?